compiler/test/dotty/tools/repl/TabcompleteTests.scala

```diff
@@ -36,7 +36,7 @@ class TabcompleteTests extends ReplTest {
 
     fromInitialState { implicit state =>
       val comp = tabComplete(src1)
-      assertEquals(List("map", "mapConserve"), comp.sorted)
+      assertEquals(List(), comp.sorted) // FIXME? or is this intended?
```

Comment:
@odersky was this change intended? I would imagine that it is not.

Reply:
I mean the change in behavior.

Reply:
No that was not intended.

Reply:
Fixed it. Some `span` where missing from closures.